File: Java/1773.java
class Solution {
    public int countMatches(List<List<String>> items, String ruleKey, String ruleValue) {
        int count = 0;
        int index = 0;
        if (ruleKey.equals("color")) {
            index = 1;
        }
        else if (ruleKey.equals("name")) {
            index = 2;
        }
        for(int i=0; i<items.size(); i++) {
            if ((items.get(i).get(index)).equals(ruleValue) ) {
                count += 1;
            }
        }
        return count;
    }
}

File: Solutions/Python/1773.py
class Solution:
    def countMatches(self, items: List[List[str]], ruleKey: str, ruleValue: str) -> int:
        count = 0
        a = ["type", "color", "name"]
        x = a.index(ruleKey)
        for i in range(len(items)):
            if ruleValue == items[i][x]:
                count += 1
        return count
